x-pack/test/functional/apps/remote_clusters/ccs/remote_clusters_index_management_flow.ts

```diff
@@ -0,0 +1,107 @@
+        await retry.waitFor('table to be visible', async () => {
+          return await testSubjects.isDisplayed('remoteClusterListTable');
+        });
+        const remotes = await pageObjects.remoteClusters.getRemoteClustersList();
+        expect(remotes.length).to.eql(1);
+        expect(remotes[0].remoteName).to.eql('ftr-remote');
+        expect(remotes[0].remoteAddress).to.contain('localhost');
+        expect(remotes[0].remoteStatus).to.eql('Connected');
+        expect(remotes[0].remoteConnectionCount).to.eql('1');
+        expect(remotes[0].remoteMode).to.eql('default');
+      });
+    });
+
+    describe('Cross Cluster Replication', function () {
+      before(async () => {
+        await remoteEs.indices.create({
+          index: 'my-index',
+          body: {
+            settings: { number_of_shards: 1, soft_deletes: { enabled: true } },
+          },
+        });
+        await pageObjects.common.navigateToApp('crossClusterReplication');
+        await retry.waitFor('indices table to be visible', async () => {
+          return await testSubjects.isDisplayed('createFollowerIndexButton');
+        });
+      });
+      it('Create Follower Index', async () => {
+        await pageObjects.crossClusterReplication.clickCreateFollowerIndexButton();
+        await pageObjects.crossClusterReplication.createFollowerIndex(
+          leaderName,
+          followerName,
+          true,
+          '1s'
+        );
+      });
+    });
+    describe('Index Management', function () {
+      before(async () => {
+        await pageObjects.common.navigateToApp('indexManagement');
+        await retry.waitForWithTimeout('indice table to be visible', 15000, async () => {
+          return await testSubjects.isDisplayed('indicesList');
+        });
+        await remoteEs.index({
+          index: 'my-index',
+          body: { a: 'b' },
+        });
+        await pageObjects.common.navigateToApp('indexManagement');
+        await retry.waitForWithTimeout('indice table to be visible', 15000, async () => {
+          return await testSubjects.isDisplayed('indicesList');
+        });
+      });
+      it('Made it this far', async () => {
+        await pageObjects.indexManagement.clickIndiceAt(0);
+        await pageObjects.indexManagement.performIndexActionInDetailPanel('flush');
+        await testSubjects.click('euiFlyoutCloseButton');
+        await pageObjects.common.navigateToApp('indexManagement');
+        await retry.waitForWithTimeout('indice table to be visible', 15000, async () => {
+          return await testSubjects.isDisplayed('indicesList');
+        });
+
+        const indicesList = await pageObjects.indexManagement.getIndexList();
+        const followerIndex = indicesList[0];
+        expect(followerIndex.indexDocuments).to.not.eql('0');
+        expect(followerIndex.indexDocuments).to.eql('1');
+      });
+    });
+
+    after(async () => {
+      await security.testUser.restoreDefaults();
```

Comment:
Should the indices created also be cleaned up?

Reply:
Fixed